### PR TITLE
Only install phantomjs when required

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- clean and build the GUI,
- load the GUI, the images should appear
- change a svg image and run `make svg-sprite`
- reload the gui, the image should have changed

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7825/

File: Makefile [hunk 1]
```diff
@@ -3,10 +3,8 @@ PY := bin/python
 PYTEST := bin/py.test
 GUISRC := jujugui/static/gui/src
 GUIBUILD := jujugui/static/gui/build
-SVG_SPRITE_DIR := $(GUIBUILD)/app/assets
-SVG_SPRITE_FILE := $(SVG_SPRITE_DIR)/stack/svg/sprite.css.svg
+SVG_SPRITE_DIR := $(GUISRC)/app/assets
```
Comment (hatched) [inline Makefile]: This is just the assets directory so you can probably update accordingly

File: Makefile [hunk 2]
```diff
@@ -188,27 +187,35 @@ $(STATIC_FONT_FILES): $(FONT_FILES)
...
 
+$(SVG_SPRITE_MODULE):
+	npm install svg-sprite@1.3.6
+
 .PHONY: images
-images: $(STATIC_IMAGES) $(SVG_SPRITE_FILE) $(FAVICON)
+images: $(STATIC_IMAGES) $(FAVICON)
+
+.PHONY: svg-sprite
+svg-sprite: $(SVG_SPRITE_MODULE)
```
Comment (hatched) [inline Makefile]: Can you add this to the help output

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7826/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7827/

**Comment by huwshimi** [comment]
Thanks @hatched and @makyo :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**